client/python/docs/source/conf.py

```diff
@@ -12,27 +12,28 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../..'))
+
+sys.path.insert(0, os.path.abspath("../.."))
```

Comment:
Please revert all changes in this file since they are noops.

Reply:
Coolio :)